altair/vegalite/v5/api.py

```diff
@@ -2657,6 +2657,32 @@ def to_dict(
             validate=validate, format=format, ignore=ignore, context=context
         )
 
+    def _transformed_data(
+        self,
+        row_limit: Optional[int] = None,
+        exclude: Optional[Iterable[str]] = None,
+    ) -> Optional[pd.DataFrame]:
+        """Evaluate a Chart's transforms
+
+        Evaluate the data transforms associated with a Chart and return the
+        transformed data a DataFrame
+
+        Parameters
+        ----------
+        row_limit : int (optional)
+            Maximum number of rows to return for each DataFrame. None (default) for unlimited
+        exclude : iterable of str
+            Set of the names of charts to exclude
+
+        Returns
+        -------
+        DataFrame
+            Transformed data as a DataFrame
+        """
+        from ...utils.transformed_data import transformed_data
```

Comment:
The existing codebase uses relative imports in many places so this would be consistent but I'm in favour of switching to absolute ones. They are easier to read, especially as Altair has many modules with the same names on different levels.